rules/signing.bzl

```diff
@@ -959,6 +959,7 @@ def _signature_test_impl(ctx):
       substitutions = {
         "@FILES@": " ".join(files),
         "@OPENTITANTOOL@": ctx.executable._opentitantool.short_path,
+        "@VERIFY_ARGS@": "--spx --domain " + ctx.attr.spx_domain,
```

Comment:
Recall that a domain of "" (empty string) means to not perform spx verification.

I'd suggest computing the verify args before the template function call:
```
verify_args = "" if not ctx.attr.spx_domain else "--spx --domain=" + ctx.attr.spx.domain
```
And then pass that as the value of `@VERIFY_ARGS@` in the substituations.